README.md

```diff
@@ -1,2 +1,83 @@
 # filter-strings-php
-A filtering implementation for verifying correct data and performing typical modifications to strings.
+
+[![Build Status](https://travis-ci.org/traderinteractive/filter-strings-php.svg?branch=master)](https://travis-ci.org/traderinteractive/filter-strings-php)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/traderinteractive/filter-strings-php/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/traderinteractive/filter-strings-php/?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/traderinteractive/filter-strings-php/badge.svg?branch=master)](https://coveralls.io/github/traderinteractive/filter-strings-php?branch=master)
+
+[![Latest Stable Version](https://poser.pugx.org/traderinteractive/filter-strings/v/stable)](https://packagist.org/packages/traderinteractive/filter-strings)
+[![Latest Unstable Version](https://poser.pugx.org/traderinteractive/filter-strings/v/unstable)](https://packagist.org/packages/traderinteractive/filter-strings)
+[![License](https://poser.pugx.org/traderinteractive/filter-strings/license)](https://packagist.org/packages/traderinteractive/filter-strings)
+
+[![Total Downloads](https://poser.pugx.org/traderinteractive/filter-strings/downloads)](https://packagist.org/packages/traderinteractive/filter-strings)
+[![Daily Downloads](https://poser.pugx.org/traderinteractive/filter-strings/d/daily)](https://packagist.org/packages/traderinteractive/filter-strings)
+[![Monthly Downloads](https://poser.pugx.org/traderinteractive/filter-strings/d/monthly)](https://packagist.org/packages/traderinteractive/filter-strings)
+
+A filtering implementation for verifying the contents of strings and some common formats of strings.
+
+## Requirements
+
+Requires PHP 7.0 or newer and uses composer to install further PHP dependencies.  See the [composer specification](composer.json) for more details.
+
+## Installation
+
+filter-floats-php can be installed for use in your project using [composer](http://getcomposer.org).
+The recommended way of using this library in your project is to add a `composer.json` file to your project.  The following contents would add filter-floats-php as a dependency:
+```sh
+composer require traderinteractive/filter-floats
+```
+
+### Functionality
+
+#### Strings::filter
+Aliased in the filterer as `string`, this filter verifies that the argument is a string.  The second parameter can be set to `true` to allow
```

Comment:
For all of these the statement
> Aliased in the filterer as `blah`
has no meaning as the filterer doesn't exist in this library